Frameworks/CoreText/DWriteWrapper.mm

```diff
@@ -47,12 +47,56 @@
     return s_dwriteFactory;
 }
 
+template <typename TElement>
+bool __CloneArray(_In_reads_opt_(count) TElement const* source,
```

Comment:
duh

---
In reply to: [78623064](https://github.com/Microsoft/WinObjC/pull/999#discussion_r78623064) [](ancestors = 78623064,78621358,78468942)